src/pre_process/m_start_up.fpp

```diff
@@ -837,7 +837,7 @@ contains
 &                pTg-equilirium (relax = "T" activated)'
             end if
 
-            call s_relaxation_solver(q_cons_vf)
+            call s_infinite_relaxation_k(q_cons_vf)
```

Comment:
is this intentional? seems wrong?

Reply:
It's intentional, function pointers don't work for phase change